course/models.py

```diff
@@ -1,3 +1,21 @@
 from django.db import models
-
+from course.choices import Status
 # Create your models here.
+
+
+class Courses(models.Model):
+    Lecturer_name = models.CharField(max_length=255, null=True, blank=True)
+    Lecturer_email = models.EmailField(max_length=255, null=True, blank=True)
+    Availability_status = models.CharField(max_length=255, choices=Status.choices, null=True, blank=True, default="")
```

Comment:
Default cannot be an empty string, we need to always know the status. And, this is the status of what, the lecturer or the course?